### PR TITLE
Add Firewall FAQ entries

### PR DESCRIPTION
**Pull request by pedrosousa**
Addresses https://jira.cfops.it/browse/PCX-1008.

### CODE REVIEW

File: products/firewall/src/content/known-issues-and-faq/index.md [hunk 1]
```diff
@@ -373,8 +368,47 @@ Block Amazon Web Services (AWS) and Google Cloud Platform (GCP) because of large
...
+No. The _Challenge (Captcha)_ and _JS Challenge_ actions only support HTML requests.
+
+Challenges presented to users display an intermediate page where they must prove they are not a bot. This concept does not work over XHR or AJAX.
+
+When an XHR or AJAX request triggers one of the _Challenge_ actions, the resulting request will have the following status code:
+
+* HTTP status code 403 for _Challenge (Captcha)_
+* HTTP status code 503 for _JS Challenge_
+
+Your application can use these status codes to handle the case of challenges being issued unexpectedly.
```
Comment (kodster28) [inline products/firewall/src/content/known-issues-and-faq/index.md]: Reword to "handle unexpected challenges."

File: products/firewall/src/content/known-issues-and-faq/index.md [hunk 2]
```diff
@@ -373,8 +368,47 @@ Block Amazon Web Services (AWS) and Google Cloud Platform (GCP) because of large
...
+
+## Does the 'challengeFailed' action accurately represent challenges that users did not pass?
+
+No. The `challengeFailed` and `jschallengeFailed` Firewall actions account for observed requests that, under special circumstances, did not pass a challenge. However, some failed challenges cannot be traced back to a Firewall rule. Additionally, the Firewall may not have a record of every request with a failed challenge.
+
+Therefore, consider these actions with caution. A reliable indicator is the CSR (Challenge Solve Rate) displayed in **Firewall Rules**, which is calculated as follows: `number of challenges solved / number of challenges issued`.
+
+## Why don't I see any failed challenges? Why is 'ChallengeIssued' not equal to 'ChallengeSolved' plus 'ChallengeFailed'?
```
Comment (kodster28) [inline products/firewall/src/content/known-issues-and-faq/index.md]: Maybe change "don't" to "would I not" to get rid of contraction?

File: products/firewall/src/content/known-issues-and-faq/index.md [hunk 3]
```diff
@@ -373,8 +368,47 @@ Block Amazon Web Services (AWS) and Google Cloud Platform (GCP) because of large
...
+
+Users do not complete all challenges. Cloudflare issues challenges that are never answered — only 2-3% of all served challenges are actually being answered.
```
Comment (kodster28) [inline products/firewall/src/content/known-issues-and-faq/index.md]: "challenges are usually answered"?

File: products/firewall/src/content/known-issues-and-faq/index.md [hunk 4]
```diff
@@ -373,8 +368,47 @@ Block Amazon Web Services (AWS) and Google Cloud Platform (GCP) because of large
...
+
+There are multiple reasons for this:
+
+* Users give up on a challenge.
+* Users try to solve a challenge but cannot provide an answer.
+* Users keep refreshing the challenge to get a challenge they can solve, and they don't submit answers.
```
Comment (kodster28) [inline products/firewall/src/content/known-issues-and-faq/index.md]: Maybe reword to "Users keep refreshing the challenge but never submit an answer."

**Review by kodster28 — APPROVED** [review]

**Review by migueldemoura — APPROVED** [review]
